### ISSUE TITLE
Error: open : no such file or directory while running `format-coverage`

### ISSUE THREAD

**Issue by mazjs** [codeclimate/test-reporter#184]
Command I am running : 

```
 ../../cc-test-reporter format-coverage -t lcov --output ../../coverage/codeclimate.{service}.json -p /services/{service} -d
```

Error I am getting: 

```
DEBU[0000] using formatter lcov                         
DEBU[0000] couldn't load git info from ENV, trying git... 
Error: open : no such file or directory
Usage:
cc-test-reporter format-coverage [flags]

Flags:
-t, --input-type string   type of input source to use [simplecov, lcov, coverage.py, clover, gocov, gcov, cobertura]
-o, --output string       output path (default "coverage/codeclimate.json")
-p, --prefix string       the root directory where the coverage analysis was performed (default "/home/rof/clone/services/delivery")

Global Flags:
-d, --debug   run in debug mode
```

Any obvious mistake I am making? 

I am trying to make this work in codeship.

**Comment by ale7714** [comment]
@mazjs thank you for trying our new test-reporter. I'm working on a fix that addresses this issue. I'll be letting you know when is released.

**Comment by ale7714** [comment]
@mazjs we just released [v0.1.10](https://github.com/codeclimate/test-reporter/releases/tag/v0.1.10) that should fix this issue. Let me know if you encounter anything else, I'm happy to help. I'm closing this for now.
